### PR TITLE
#478 Constructors should be code free

### PR DESCRIPTION
**Pull request by HDouss**
Resolving #478 Constructors should be code free.
Added XPath rule and corresponding unit test.

### CODE REVIEW

**Comment by davvd** [comment]
@hdouss Many thanks for the PR, let me find a reviewer for it

**Comment by davvd** [comment]
@pinaf it's yours, please review

**Comment by HDouss** [comment]
@pinaf Could you please review my PR ?

File: qulice-pmd/src/test/resources/com/qulice/pmd/CallToConstructorInConstructor.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package foo;
+
+public final class CallToConstructorInConstructor extends Super{
```
Comment (pinaf) [inline qulice-pmd/src/test/resources/com/qulice/pmd/CallToConstructorInConstructor.java]: @HDouss let's add a whitespace before the ending `{`

**Comment by pinaf** [comment]
@HDouss nice. 1 minor comment.

**Comment by pinaf** [comment]
@HDouss sorry about the delay - holidays.

**Comment by HDouss** [comment]
@pinaf Thanks. Done!

**Comment by HDouss** [comment]
@pinaf sorry. Have fun. Merry Christmas and Happy new year!

**Comment by pinaf** [comment]
@rultor merge

**Comment by pinaf** [comment]
@HDouss thanks! the same to you!

**Comment by krzyk** [comment]
@rultor merge pls

**Comment by pinaf** [comment]
@rultor deploy

**Comment by krzyk** [comment]
@rultor deploy

**Comment by davvd** [comment]
@pinaf done, I added **21 mins** in payment `AP-0FJ13717MH2103428`, 95 hours and 48 mins spent to complete this

extra minutes for review comments (c=6)

added +21 to your rating, now it is equal to [+4880](http://www.netbout.com/b/35128?open=rating)

**Comment by davvd** [comment]
@rultor please deploy

**Pull request merged**